server/homebrew.api.js

```diff
@@ -33,19 +72,36 @@ const MAX_TITLE_LENGTH = 100;
 
 const getGoodBrewTitle = (text)=>{
 	const tokens = Markdown.marked.lexer(text);
- 	return (tokens.find((token)=>token.type == 'heading' ||	token.type == 'paragraph')?.text || 'No Title')
+ 	return (tokens.find((token)=>token.type === 'heading' || token.type === 'paragraph')?.text || 'No Title')
 				 .slice(0, MAX_TITLE_LENGTH);
 };
 
 const excludePropsFromUpdate = (brew)=>{
 	// Remove undesired properties
-	const propsToExclude = ['views', 'lastViewed'];
+	const propsToExclude = ['_id', 'views', 'lastViewed', 'editId', 'shareId', 'googleId'];
 	for (const prop of propsToExclude) {
 		delete brew[prop];
 	}
 	return brew;
 };
 
+const excludeStubProps = (brew)=>{
+	const propsToExclude = ['text', 'textBin', 'renderer', 'pageCount', 'version'];
+	for (const prop of propsToExclude) {
+		brew[prop] = undefined;
+	}
+	return brew;
+};
+
+const excludeGoogleProps = (brew)=>{
+	const modified = brew.toObject ? brew.toObject() : brew;
```

Comment:
Is this a symptom of not knowing if `brew` is a normal object or a Mongo document?

Reply:
Yep -- google brews are normal objects, stub/HB brews are mongoose. I'm pulling that out with my updates